logger/src/lib.rs

```diff
@@ -786,7 +783,7 @@ impl Log for Logger {
         if self.enabled(record.metadata()) {
             let msg = format!(
                 "{}{}{}{}",
-                Local::now().format(TIME_FMT),
+                LocalTime::now(),
```

Comment:
Is the format of the log entries changed? If yes, than you should also update the CHANGELOG.md file and specify that the format is changed.

Reply:
The format is the same